attributes/default.rb

```diff
@@ -71,6 +71,15 @@
   'GSSAPIAuthentication' => 'no'
 }
 
+# sshd_config permissions
+default['sshd']['sshd_config']['mode'] =
```

Comment:
The `default['sshd']['sshd_config']` mode attribute is actually used as a hash directly to compose the `sshd_config` file directly (see [here](https://github.com/chr4-cookbooks/sshd/blob/master/definitions/openssh_server.rb#L54)). This would mess up the config file. Can you change the attribute to `default['sshd']['sshd_config_mode']`?